firmware/boron-ins-fsm/src/imDoorSensor.h

```diff
@@ -39,13 +40,22 @@ typedef struct IMDoorID {
     unsigned char byte3;
 } IMDoorID;
 
+typedef enum
+{
+    INTERNAL,
+    EXTERNAL
+} antennaType;
```

Comment:
Reading this value later in the file, I found it difficult to identify that `antennaType` was a datatype because its values are the types of antennas. 

What do you think about using the `_t` convention? So this would be something like `antennaType_t`?

Reply:
perhaps it's better to do away with this enum altogether. I thought it would make the code more readable, but I think that can be accomplished by changing the variable to `bool useExternalBLEAntenna`

Reply:
Sure! A boolean makes sense to me and would probably require less new code.